### ISSUE TITLE
UX: random hub link should not open Hub

### ISSUE THREAD

**Issue by larssondenise** [alkem-io/alkemio#1046]
## Description
As a UX lead, I don't want users to be able to see a fake private Hub when they type in a random name

## Acceptance criteria

- [ ] Decide on strategy /design for such cases

## Additional Context
Idea is to let people go to the home page with a notification doesn't exist
![image.png](https://images.zenhubusercontent.com/6200e10a5561c893d9e84591/4740e883-33c4-4b97-bdbd-b1396d92c3a7)

**Comment by techsmyth** [comment]
@SimoneZaza @larssondenise maybe the client should do a check to see if it can retrieve a hub using the potential nameID + if that fails then is redirects to home page with an error message?

**Comment by techsmyth** [comment]
@me-andre fair point. We would then need a clean "content not found" page that is shown which we can make "nice", e.g. with a link to home and the option to reach out for support?
